core/src/main/java/org/elasticsearch/search/suggest/phrase/LinearInterpoatingScorer.java

```diff
@@ -33,8 +33,8 @@
     private final double bigramLambda;
     private final double trigramLambda;
 
-    public LinearInterpoatingScorer(IndexReader reader, Terms terms, String field,  double realWordLikelyhood, BytesRef separator, double trigramLambda, double bigramLambda, double unigramLambda)
-            throws IOException {
+    public LinearInterpoatingScorer(IndexReader reader, Terms terms, String field, double realWordLikelyhood, BytesRef separator,
```

Comment:
we have to rename this class. s/interpoating/interpolating   as a followup. Let's not forget :)